### PR TITLE
Added env variable

### PR DESCRIPTION
**Pull request by spathare21**
##### ISSUE TYPE

 - Feature Pull Request
 - Bug fix Pull Request
 - Docs Pull Request

### CODE REVIEW

**Comment by spathare21** [comment]
@sundeep-co-in by mistake created this PR was wanted just change the env variable in dockerfile but looks diff here. unable to delete this PR  please delete if possible.

**Comment by sundeep-co-in** [comment]
@spathare21 Thanks for the PR, I think `git pull --rebase` may fix this issue. 

I like the idea of env variables. Perhaps we need to do it for app as well. Transtats app seeks `keys.json` for values to talk with db. So, along with `launch.sh` we need to tell django as well to pick values from env variables. Better if we make django app first to look for env variables if not found then it may look for `keys.json` file. Thereby this could be implemented.

@pnemade I would request your views on this!

File: deploy/docker/Dockerfile [hunk 1]
```diff
@@ -6,21 +6,31 @@
 FROM fedora
 MAINTAINER Sundeep Anand <suanand@redhat.com>
 
+
+# Environment variable 
+ENV DATABASE_NAME=transtats DATABASE_USER=postgres DATABASE_PASSWD=postgres DATABASE_HOST=localhost TRANSIFEX_USER=suanand TRANSIFEX_PASSWD=transtats YTHONUNBUFFERED=1
```
Comment (sundeep-co-in) [inline deploy/docker/Dockerfile]: `YTHONUNBUFFERED` should be `PYTHONUNBUFFERED`

Comment (pnemade) [inline deploy/docker/Dockerfile]: Good to keep lines limited to 80 characters. Split long lines.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Dockerfile for transtats
+# http://transtats.org/
+#
+# Run Command: cd /transtats
```
Comment (sundeep-co-in) [inline Dockerfile]: May be two: `build` and `run` comments would be more verbose!

```
# Build Image: docker build -t transtats .
# Run Container: docker run -d --name containter -p 8080:8015 transtats
```

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+# docker build -t transtats .
+# docker run -d --name container -p 8080:8015 transtats
```
Comment (sundeep-co-in) [inline Dockerfile]: Include `env` variables example

File: deploy/docker/Dockerfile [hunk 4]
```diff
@@ -1,27 +1,44 @@
 # Dockerfile for transtats
```
Comment (sundeep-co-in) [inline deploy/docker/Dockerfile]: we may keep this `Dockerfile` as is. README.md may contain info to build `Dockerfile with env vars`

File: deploy/docker/launch.sh [hunk 5]
```diff
@@ -4,11 +4,23 @@ cd /workspace
...
+if [ -z "$DATABASE_USER" ]
+then
+    echo "Setting database name, user and pass"
+    export DATABASE_USER="postgres"
+    export DATABASE_PASSWD="postgres"
+    export DATABASE_NAME="transtats"
+else
+    echo "Using Database name ,user & pass from env variable set in dockerfile"
+fi
+    sudo -u postgres psql -c "ALTER USER $DATABASE_USER WITH PASSWORD '$DATABASE_PASSWD';"
```
Comment (sundeep-co-in) [inline deploy/docker/launch.sh]: Is indentation required for below four lines?

**Comment by spathare21** [comment]
Created new PR https://github.com/transtats/transtats/pull/58

**Pull request closed without merging**